bundles/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/FreeboxBinding.java

```diff
@@ -222,42 +251,162 @@ protected void execute() {
 	@SuppressWarnings("incomplete-switch")
 	@Override
 	protected void internalReceiveCommand(String itemName, Command command) {
+		if ((loginManager == null) || (!loginManager.isConnected())) {
+			try {
+				authorize();
+			} catch (FreeboxException e) {
+				logger.error(e.getMessage());
```

Comment:
please decrease severity to info since this exception does not effect the stability of openHAB itself